tests/unit/controllers/UserManagementTest.php

```diff
@@ -110,6 +110,7 @@ public function testUpdateAdminUserFullName() {
 
     public function testUpdateAdminUserTamperproofed()
     {
+        $_SESSION = [];
```

Comment:
We don't have already a test helper for this ?

Reply:
To clear session? Maybe?

Reply:
Seems not (quick look)